modules/apps/foundation/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/npm/NPMRegistry.java

```diff
@@ -45,27 +45,52 @@
 import org.osgi.util.tracker.BundleTrackerCustomizer;
 
 /**
+ * <p>
+ * This is the central class for the NPM packages support of Liferay Portal.
+ * </p>
+ * <p>
+ * It features a central registry where all NPM packages and modules deployed
+ * accross OSGi bundles are tracked.
+ * </p>
```

Comment:
Yes. It must be an error, I guess. Should say 'with' or something like 'inside'. 👍